### DISCUSSION TITLE
the database field type is varchar and it is defined as String in Schema, an error occurs because it is returned in Btyes format when retrieving data.

### DISCUSSION THREAD

**Discussion by usvillage-source** [prisma/prisma#19532] [Q&A]
prisma version: 4.1.5
mariadb version: 10

the database field type is varchar and it is defined as String in Schema, an error occurs because it is returned in Btyes format when retrieving data.

If i write the schema in byte format it works fine

code: 'P2032',
  clientVersion: '4.15.0',
  meta: {
    field: 'userId',
    expected_type: 'String',
    found: 'c2VvMDc0ND09PWI2YmJjYzVh'
  },

**Comment by ludralph** [comment]
Hi @usvillage-source 👋 

Can you please share the relevant parts of your schema?

**Reply by nurul3101** [reply]
@usvillage-source Did you had a chance to look at Raphael's last message?

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
